src/app/recents/recents.component.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { PropertyDisplayPipe, PropertyValueDisplayPipe } from '../pipes/propertydisplay';
+
+
+fdescribe('RecentsComponent', () => {
+  let fixture: ComponentFixture<RecentsComponent>;
+  let recentsService: RecentsService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        ArtifactsTableComponent,
+        EstuaryTableComponent,
+        NodeTypeDisplayPipe,
+        NodeExternalUrlPipe,
+        PropertyDisplayPipe,
+        PropertyValueDisplayPipe,
+        RecentsComponent,
+        TruncatePipe
+      ],
+      providers: [
+        RecentsService,
+        DatePipe
+      ],
+      imports: [
+        TabsModule.forRoot(),
+        HttpClientTestingModule,
+        NoopAnimationsModule,
+        BsDropdownModule.forRoot()
+      ],
+      // This is used to avoid having to import components we don't test here
+      schemas: [NO_ERRORS_SCHEMA]
+    }).compileComponents();
+    recentsService = TestBed.get(RecentsService);
+  });
+
+  it('should retrieve the 5 most recent advisories', fakeAsync(() => {
+    spyOn(recentsService, 'getRecents').and.returnValue(
+      of(recents)
+    );
+    fixture = TestBed.createComponent(RecentsComponent);
+    fixture.detectChanges();
+    const component = fixture.componentInstance;
+    expect(component.recentsAll).toBe(<any> recents.data);
+    const tabs = fixture.debugElement.queryAll(By.css('tab'));
+    expect(tabs.length).toBe(5);
+
+    // Ensure the tab headings are correct
+    expect(tabs[0].nativeElement.getAttribute('ng-reflect-heading')).toBe('Advisory');
+    expect(tabs[1].nativeElement.getAttribute('ng-reflect-heading')).toBe('Commit');
+    expect(tabs[2].nativeElement.getAttribute('ng-reflect-heading')).toBe('Freshmaker Event');
+    expect(tabs[3].nativeElement.getAttribute('ng-reflect-heading')).toBe('Build');
+    expect(tabs[4].nativeElement.getAttribute('ng-reflect-heading')).toBe('Bugzilla Bug');
+
+    // Ensure that table headers are correct
+    const tableHeaders = fixture.debugElement.queryAll(By.css('th'));
+    expect(tableHeaders[0].nativeElement.textContent.trim()).toBe('Advisory Name');
+    expect(tableHeaders[1].nativeElement.textContent.trim()).toBe('ID');
+    expect(tableHeaders[2].nativeElement.textContent.trim()).toBe('Security Impact');
+    expect(tableHeaders[3].nativeElement.textContent.trim()).toBe('State');
+    expect(tableHeaders[4].nativeElement.textContent.trim()).toBe('Synopsis');
+    expect(tableHeaders[5].nativeElement.textContent.trim()).toBe('Update Date');
+
+    // Ensure the correct content is being displayed after clicking a new tab
+    const tabContent = fixture.debugElement.queryAll(By.css('.nav-link'));
+    tabContent[1].nativeElement.click();
+    tick(1000);
+    fixture.detectChanges();
+    const activeTab = fixture.debugElement.query(By.css('.nav-item .active'));
+    expect(activeTab.nativeElement.id).toBe('DistGitCommit-link');
```

Comment:
How about checking the `textContent` instead?

Also, could you verify that the table updated as well? Checking the table headers again like you did before would suffice.